tests/scripts/thread-cert/node.py

```diff
@@ -522,6 +522,13 @@ def remove_domain_prefix(self, prefix):
         self.remove_prefix(prefix)
         self.register_netdata()
 
+    def set_next_dua_response(self, status, iid=None):
+        cmd = 'bbr mgmt dua {}'.format(status)
+        if iid is not None:
+            cmd += ' ' + str(iid)
```

Comment:
check if `iid` is valid to catch error ?

Reply:
it might be not necessary? as cli command will take the parameter and output parse error if invalid.